src/objects/nodes.ts

```diff
@@ -78,8 +109,8 @@ async function createNode(node: RunGraphNode): Promise<NodeSprites> {
     label,
   })
 
-  viewport.addChild(graphics)
-  viewport.addChild(label)
+  container!.addChild(graphics)
+  container!.addChild(label)
```

Comment:
why the non-null assertion? is this coming from pixi and not well-typed?

Reply:
oh I forgot to come back to this. I didn't care to fix it until I knew it was working. Will update

Reply:
Fixed!